cmd/plugins/topology-aware/policy/resources.go

```diff
@@ -998,19 +1026,19 @@ func (cr *request) String() string {
 	isolated := map[bool]string{false: "", true: "isolated "}[cr.isolate]
 	switch {
 	case cr.full == 0 && cr.fraction == 0:
-		return fmt.Sprintf("<CPU request "+cr.container.PrettyName()+": ->") + mem
+		return fmt.Sprintf("<%s CPU request "+cr.container.PrettyName()+": ->", cr.prio) + mem
```

Comment:
Hmm... maybe. If any of the concatenated strings could come from an untrusted/external source (so it could contain formatting verbs) then it would be a bad idea to use both concatenation and `Sprintf`. It's not the case here, but I'll update it anyway (and all the other cases in the same function) to only use `Sprintf()` without concatenation.